wsd/DocumentBroker.cpp

```diff
@@ -2624,9 +2636,9 @@ void DocumentBroker::setLoaded()
         _docState.setLive();
         _loadDuration = std::chrono::duration_cast<std::chrono::milliseconds>(
                                 std::chrono::steady_clock::now() - _threadStart);
-        const auto minTimeoutSecs = ((_loadDuration * 4).count() + 500) / 1000;
-        _saveManager.setSavingTimeout(
-            std::max(std::chrono::seconds(minTimeoutSecs), std::chrono::seconds(5)));
+        const auto minTimeoutSecs = std::chrono::duration_cast<std::chrono::seconds>(
+            _loadDuration * 4 + std::chrono::milliseconds(500));
```

Comment:
```suggestion
            _loadDuration) * 4 + std::chrono::milliseconds(500);
```

Reply:
Care to elaborate or suggest a better math? It's just a typical ratio between load and save, so if a document takes long to load we also wait long on the save.

I'm open to suggestions for better math.

Reply:
Btw, the above will simply change the timeout to milliseconds. Was that intentional?

Notice that I'm rounding. If that's what you mean is fishy, that's actually pretty standard rounding-up math.

Reply:
After offline discussion, the updated version should prove better than the original.